server/utils/validation.ts

```diff
@@ -72,6 +72,20 @@ export const errorMessage = (field: string, text: string): ErrorMessage => {
   }
 }
 
+export const insertPropertyError = (error: SanitisedError | Error, propertyName: string, errorType: string): void => {
+  const data = 'data' in error ? error.data : {}
+  const invalidParams = (data['invalid-params'] ? data['invalid-params'] : []) as Array<Record<string, string>>
+
+  invalidParams.push({
+    propertyName: `$.${propertyName}`,
+    errorType,
+  })
+
+  data['invalid-params'] = invalidParams
+  // eslint-disable-next-line dot-notation
```

Comment:
Is there a reason we can't use dot notation here?

Reply:
Then it fails on the typecheck step, as `data` isn't a known part of the Error type